README.md

```
@@ -62,6 +62,47 @@ The command responds with the number of tokens left in the bucket.
     127.0.0.1:6379> SHIELD.absorb user123 30 60 13
     (integer) -1
 
+## Development
+
+### Running Tests
+
+```bash
```

markdownlint-cli2 (0.18.1)

Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)